### ISSUE TITLE
Sidebar is constrained after restart

### ISSUE THREAD

**Issue by ianb** [mozilla/side-view#274]
The report is [described here](https://www.reddit.com/r/firefox/comments/8or6md/test_pilot_introducing_firefox_color_and_side_view/e05rq1u/?context=3)

**Comment by dbaaz** [comment]
Hi, I'm guy that posted this on reddit.

To clarify, the bug didn't start after I restarted the browser, the sidebar was constrained the moment I installed the addon. However, restarting the browser made the sidebar even more constrained. 

When I first installed it, I was able to expand it to about 30-35% (just an estimation) of the total screen width, now its much less (20ish %) as shown in the video on my post.

**Comment by Softvision-CristinaBadescu** [comment]
Hello,

Thanks for reporting this. I tested this issue on latest Nightly build 62.0a1, latest Firefox Developer Edition 61.0b11 (Build ID: 20180604143143)  and managed to reproduce it, but only if I have the "Dark Mode" addon installed and restarted the browser. 

This issue is not reproducible on Firefox Release 60.0.1. Considering this, I've ran the Mozregression tool and managed to find a regression window. 
I've filed a platform issue here: [1467113](https://bugzilla.mozilla.org/show_bug.cgi?id=1467113).
